scm-ui/ui-webapp/src/repos/codeSection/components/CodeActionBar.tsx

```diff
@@ -71,6 +75,7 @@ const CodeActionBar: FC<Props> = ({ selectedBranch, branches, onSelectBranch, sw
             />
           )
         }
+        children={actions}
```

Comment:
I find it easier to understand that way. A `Level` has a left, right and child part in the middle. The props reflect this in this exact order. Naively, you might otherwise wonder why only the child prop is passed differently, even though all three types are 'equivalent'. If only one child was handed over, I would agree with you immediately.

`<div className="level">
<div className="level-left">{left}</div>
{child}
<div className="level-right">{right}</div>
</div>`